### ISSUE TITLE
Accessibility Audit: Unique label is not associated with See Detail links on Listings pages

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#957]
Turn on the Screen Reader, and go to the [Listings](https://homeconnect.detroitmi.gov/listings) page. Navigate to each See Details link associated with the listings on the page, and observe the announcement. The Screen reader announces 'See Details' link for all the different listing on the page. It will be confusing if the user wants to skim content by links. Ideally, we would add a link label that would be descriptive enough so that user can understand the context even without the surrounding text (e.g. 'See Details for [property name]').

https://user-images.githubusercontent.com/394908/148579305-900e7e33-3db9-4db8-8721-428da831ac93.mov

**Comment by emilyjablonski** [comment]
Fixed in Seeds
